src/views/auth/login.pug

```diff
@@ -0,0 +1,43 @@
+doctype html
+html
+  head
+    title= title
+    meta(charset='UTF-8')
+    meta(name='viewport', content='width=device-width, initial-scale=1.0')
+    link(rel='stylesheet', href='/stylesheets/login.css')
+    link(rel='stylesheet', href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css')
+  
+  body
+    .background
+      .login-container
+        img.logo(src='/favicon.ico', alt='Logo')
+        h2 Welcome to SmartEdu!
```

Comment:
I18n toàn bộ static text trong file view e nhé

Reply:
dạ pull này chỉ có layout thoi ạ